babel.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function conf(api) {
+  const presets = [
+    ['@babel/preset-env', { modules: false }],
+    '@babel/preset-react',
+    '@babel/preset-typescript',
+  ];
+  const plugins = [];
+
+  api.cache(true);
```

Comment:
❗️
Давай лучше как в доке напишем, сначала работу с переданным `api`, а потом наши расширялки. В этом не очень много смысла, но чем ближе к доке, тем лучше и "конвенциональнее"
```suggestion
  api.cache(true);
  
  const presets = [
    ['@babel/preset-env', { modules: false }],
    '@babel/preset-react',
    '@babel/preset-typescript',
  ];
  const plugins = [];
```